型定義の import に `type` を書いていく

どうも `type` がないとエラーが出るらしいので修正。先に言ってくれ
```
error - ./components/meta.tsx:3:0
Module not found: Can't resolve 'localComponentSpecs'
  1 | import React from 'react';
  2 | import Head from 'next/head';
> 3 | import { meta } from 'localComponentSpecs';
  4 |
  5 | const Meta: React.VFC<meta> = (props: meta) => {
  6 |   const title = props.title || 'huequica.xyz';
```

--- a/bulmaElements/column.tsx
+++ b/bulmaElements/column.tsx
@@ -1,5 +1,5 @@
 import React from 'react';
-import { column } from 'bulmaElementSpecs';
+import type { column } from 'bulmaElementSpecs';
 import is from 'viewLogics/is';
 
 /**

--- a/bulmaElements/elements/box.tsx
+++ b/bulmaElements/elements/box.tsx
@@ -1,5 +1,5 @@
 import React from 'react';
-import { box } from 'bulmaElementSpecs';
+import type { box } from 'bulmaElementSpecs';
 /**
  * Element contain box component
  * @see https://bulma.io/documentation/elements/box/

--- a/bulmaElements/elements/iconText.tsx
+++ b/bulmaElements/elements/iconText.tsx
@@ -1,5 +1,5 @@
 import React from 'react';
-import { iconTextInButton, iconText } from 'bulmaElementSpecs';
+import type { iconTextInButton, iconText } from 'bulmaElementSpecs';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import is from 'viewLogics/is';
 

--- a/bulmaElements/elements/title.tsx
+++ b/bulmaElements/elements/title.tsx
@@ -1,5 +1,5 @@
 import React from 'react';
-import { title } from 'bulmaElementSpecs';
+import type { title } from 'bulmaElementSpecs';
 import is from 'viewLogics/is';
 
 /**

--- a/bulmaElements/layout/hero.tsx
+++ b/bulmaElements/layout/hero.tsx
@@ -1,5 +1,5 @@
 import React from 'react';
-import { hero, heroParts } from 'bulmaElementSpecs';
+import type { hero, heroParts } from 'bulmaElementSpecs';
 import is from 'viewLogics/is';
 
 /** Hero classes decider

--- a/components/intent.tsx
+++ b/components/intent.tsx
@@ -1,5 +1,5 @@
 import React from 'react';
-import { intent } from 'localComponentSpecs';
+import type { intent } from 'localComponentSpecs';
 import is from 'viewLogics/is';
 
 const Intent: React.VFC<intent> = ({ children, link, color }: intent) => {

--- a/components/layout.tsx
+++ b/components/layout.tsx
@@ -1,5 +1,5 @@
 import React from 'react';
-import { layout } from 'localComponentSpecs';
+import type { layout } from 'localComponentSpecs';
 import NavBar from 'components/navbar';
 
 const Layout: React.VFC<layout> = ({ children, disableMargin }: layout) => {

